compiler/il/AliasSetInterface.hpp

```diff
@@ -154,15 +166,15 @@ class TR_AliasSetInterface {
       {
       TR::Compilation *comp = TR::comp();
       LexicalTimer t("removeAliases", comp->phaseTimer());
-      static_cast<AliasSetInterface*>(this)->setAliases_impl(aliasesToModify, false,  _isDirectCall, _includeGCSafePoint);
+      (this)->setAliases_impl(aliasesToModify, false,  _isDirectCall, _includeGCSafePoint);
       }
 
    void
    addAliases(TR::SparseBitVector &aliasesToModify)
       {
       TR::Compilation *comp = TR::comp();
       LexicalTimer t("addAliases", comp->phaseTimer());
-      static_cast<AliasSetInterface*>(this)->setAliases_impl(aliasesToModify, true,  _isDirectCall, _includeGCSafePoint);
+      (this)->setAliases_impl(aliasesToModify, true,  _isDirectCall, _includeGCSafePoint);
```

Comment:
can we loose the ()s around `this` since there is no cast left and they serve no purpose?

Reply:
Remove the `(this)->` in 091c9f0